trulens_eval/examples/snowflake_demo/custom_feedback/small_local_models.py

```diff
@@ -32,13 +32,9 @@ def context_relevance(
         model = self.context_relevance_model
         with torch.no_grad():
             logit = model.forward(
-                torch.tensor(
-                    tokenizer.convert_tokens_to_ids(
-                        tokenizer.tokenize(f"{question} [SEP] {context}")
-                    )
-                ).reshape(1, -1)
-            ).logits.numpy()
-            if logit.size != 1:
-                raise ValueError("Unexpected number of results from model!")
-            logit = float(logit[0, 0])
+                **tokenizer(
+                    f"premise: {question}[SEP] hypothesis: {context}", 
```

Comment:
late to the party here, but why are premise/hypothesis prefixes the right ones to use for a relevance model?

Reply:
It's how Corey trained the model so that's why he needs to do it for inference.